{"name": "test_delete_user", "status": "failed", "statusDetails": {"message": "AssertionError: Expected status code 204, but got 404", "trace": "api_setup = <utils.service_api.ServiceAPINew.ServiceAPI object at 0x000001E01C136FF0>, api_url = 'https://companydev.jobeze.com/backend/user-info'\n\n    def test_delete_user(api_setup, api_url):\n        \"\"\"Test the DELETE request for a specific user ID.\"\"\"\n        user_id = 123  # Replace with the actual user ID you want to test\n        delete_url = f\"{api_url}/users/{user_id}\"  # Use the same user ID for DELETE\n        response_delete = api_setup.delete_service_response(delete_url)\n>       api_setup.validate_response(response_delete, expected_status=204)\n\ntestsuites\\api\\test_service_api.py:58: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nself = <utils.service_api.ServiceAPINew.ServiceAPI object at 0x000001E01C136FF0>, response = <Response [404]>, expected_status = 204\n\n    @allure.step(\"Validating API response\")\n    def validate_response(self, response, expected_status=200):\n        \"\"\"Validate the API response with the expected status.\"\"\"\n        assert response is not None, \"No response received from the API\"\n>       assert response.status_code == expected_status, \\\n            f\"Expected status code {expected_status}, but got {response.status_code}\"\nE       AssertionError: Expected status code 204, but got 404\n\nutils\\service_api\\ServiceAPINew.py:89: AssertionError"}, "description": "Test the DELETE request for a specific user ID.", "steps": [{"name": "Sending DELETE request to API", "status": "passed", "parameters": [{"name": "api_url", "value": "'https://companydev.jobeze.com/backend/user-info/users/123'"}], "start": 1746169216775, "stop": 1746169217248}, {"name": "Validating API response", "status": "failed", "statusDetails": {"message": "AssertionError: Expected status code 204, but got 404\n", "trace": "  File \"C:\\Internal\\automation-framework-master\\venv\\Lib\\site-packages\\allure_commons\\_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"C:\\Internal\\automation-framework-master\\utils\\service_api\\ServiceAPINew.py\", line 89, in validate_response\n    assert response.status_code == expected_status, \\\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n"}, "parameters": [{"name": "response", "value": "<Response [404]>"}, {"name": "expected_status", "value": "204"}], "start": 1746169217248, "stop": 1746169217248}], "attachments": [{"name": "log", "source": "0a6ee2f8-e766-4aa9-91a9-b45f684a7081-attachment.txt", "type": "text/plain"}], "start": 1746169216775, "stop": 1746169217248, "uuid": "23a53c2a-fa7c-459c-90fa-593eb857d7a0", "historyId": "cd0f2404e0f43dc759c95a4d5b891204", "testCaseId": "cd0f2404e0f43dc759c95a4d5b891204", "fullName": "testsuites.api.test_service_api#test_delete_user", "labels": [{"name": "parentSuite", "value": "testsuites.api"}, {"name": "suite", "value": "test_service_api"}, {"name": "host", "value": "DESKTOP-BR6BRT3"}, {"name": "thread", "value": "34176-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "testsuites.api.test_service_api"}]}